File: codes/matrices.c
#include <stdio.h>

int main() {
    int mat_1 [][3] = {
        {1,4,6},
        {2,0,5},
        {8,3,3}
    };
    int mat_2 [3][3], i, j, total;
    
    for (i = 0; i < 3; i++) {
        for (j = 0; j < 3; j++) {
            mat_2 [i][j] = i+j;
        }
    }

    total = 0;
    for (i = 0; i < 3; i++) {
        for (j = 0; j < 3; j++) { 
            total += mat_1 [i][j] + mat_2 [i][j];
        }
    }
    printf("Total: %d\n", total);

    return 0;
}

File: output/matrices.py
def main():
	mat_1 = [[1, 4, 6], [2, 0, 5], [8, 3, 3]]
	mat_2 = [[None] * 3 for i in range(3)] 
				
	for i in range(0,3):		
		for j in range(0,3):			
			mat_2[i][j] = i+j
			
		
	total = 0
	for i in range(0,3):		
		for j in range(0,3):			
			total += mat_1[i][j]+mat_2[i][j]
			
		
	print("Total: %d\n" % (total))
	return 0
	
if __name__ == '__main__':
	main()
